mlflow/server/js/src/experiment-tracking/components/App.js

```diff
@@ -64,7 +64,7 @@ class App extends Component {
                 </NavLink>
               </div>
               <div className='header-links'>
-                <a href={'https://github.com/mlflow/mlflow'}>
+                <a href={'https://github.com/coutureai/mlflow'}>
```

Comment:
Maybe just keep the url as https://couture.ai

Reply:
The link for the text is 'GitHub', should I change it to "Couture.ai" or something else?

Reply:
change it to couture.ai as well

Reply:
Sure

Reply:
Corrected in fe4c4f7